app/app/signUp/page.jsx

```diff
@@ -0,0 +1,71 @@
+"use client"
+
+import Button from "@/components/Button";
+import {style} from "@/app/style";
+import {useRouter} from 'next/navigation';
+
+export default function SignUpPage({onClick}) {
+    const router = useRouter();
+
+    const handlerSignIn = () => {
+        router.push("/signIn");
+    }
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+
+        console.log("Form submitted");
+    }
+
+    return (
+        <section>
+            <div className="max-w-3xl mx-auto rounded-2xl bg-white shadow-xl p-7">
+                <h1 className={`${style.heading} text-center`}>Tilmeld dig</h1>
```

Comment:
There is a cool library to concat classes called `clsx`: https://www.npmjs.com/package/clsx

When conditions are easy (like in this case) -- it is easy to concatenate classes, but for more complicated ones it is a nice tool to have